app/main.py

```diff
@@ -16,4 +16,25 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+variables = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins,
+]
+for sort_variable in variables:
+    if isinstance(sort_variable, (list, dict, set)):
+        sorted_variables["mutable"].append(sort_variable)
+    else:
+        sorted_variables["immutable"].append(sort_variable)
```

Comment:
You just need to separate variables in dictionary manually. Example: 
sorted_variables["mutable"] = [list] and so on. No need to do it automatically.